bab.py

```diff
@@ -0,0 +1,28 @@
+import time
+def bab(x):
+    b=[]
+    c=[]
+    z=0
+    y=int(input())
+    start=time.time()
+    for i in range(len(x)):
+        if x[i]>y:
+            b.append(i)
+            c.append(x[i])
+            z+=1
+            if z==3:
+                break
```

Comment:
break zwykle nie jest najlepszym rozwiązaniem, jeśli musisz go użyć to wiedz że coś się dzieje. Nie często powinieneś go rzeczywiście musieć używać.